Auth Service/controllers/adminControllers.js

```diff
@@ -0,0 +1,19 @@
+const poolPromise = require('../config/pool');
```

Comment:
I feel like this controller would be in the business logic? If we were to handle the admin authentication here it would be a middleware to check if he's the admin and user. That's what I think?
But I see you haven't used this controller.
Should we keep it?